_posts/components/2018-4-1-list-with-icon.html

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "List of items"
+category: components
+description: For having a list of items with an icon in the left side.
+             If this is integrated in body of a card, then add "list-group-item-integrated.border-left-0.border-right-0" 
+             classes for removing duplicate borders to the "list-group-item" div
+---
+
+%p
```

Comment:
Are these 2 `<p>` needed?

Reply:
Are needed to separate the visual examples displayed. Aren't needed for using the code example... the main problem is that the code you add is used for showing the visual part and to show the code